### DISCUSSION TITLE
How to set a offset for the category axis?

### DISCUSSION THREAD

**Discussion by alwgnr** [GSI-CS-CO/chart-fx#434] [Q&A]
I have implemented a so-called dotted chart with the chart-fx library. Thank you for this great library.
Now I have a problem that the dots of the first category are not fully shown on the graph. How can I set a offset for the CategoryAxis so that the first category line is not the same as the zero line.
![dottedChartIssue](https://user-images.githubusercontent.com/2658682/122988227-ca2bde00-d3a1-11eb-9843-6de09ea8cc3c.png)

**Comment by RalphSteinhagen** [comment]
@alzorin unfortunately, the size of the marker (unlike the error bars/wiskers) is not taken into account when automatically computing the axis range. 

However, you can set the vertical axis range manually via `Axis::setAutoRanging(false)` and `Axis::set(min, max)`. 

Alternatively -- if you need auto-ranging -- you could add a transparent DataSet with two points that correspond to you min/max of your initial DataSet plus the extra margins needed for the marker sizes.

There is also the `Axis::autoRangePaddingProperty()` although there is a preference to clamp to '0' which you may overwrite (advanced option) via `DefaultNumericAxis::clampBoundToZero(..)`.

Hope this helps.
